<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Pref::TInput</class>
 <widget class="QWidget" name="Gui::Pref::TInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>576</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="input_tabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="keyboard_tab">
      <attribute name="title">
       <string>&amp;Keyboard</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>16</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>16</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="keyboard_icon">
           <property name="text">
            <string notr="true">icon</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="actioneditor_desc">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Here you can change the shortcuts for actions. Double click an action to change its shortcut. You can save the list for later use, sharing or transport to another computer.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="action_editor_layout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="Gui::Action::TActionsEditor" name="actions_editor" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mouse_tab">
      <attribute name="title">
       <string>&amp;Mouse</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="margin">
        <number>12</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="mouse_icon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string notr="true">IconIconIcon</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="button_group">
           <property name="title">
            <string>Buttons</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="7" column="2">
             <widget class="QComboBox" name="xbutton1_click_combo"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="left_click_label">
              <property name="text">
               <string>&amp;Left click</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>left_click_combo</cstring>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="Gui::Pref::TComboBox" name="right_click_combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="right_click_label">
              <property name="text">
               <string>&amp;Right click</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>right_click_combo</cstring>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QComboBox" name="xbutton2_click_combo"/>
            </item>
            <item row="0" column="2">
             <widget class="Gui::Pref::TComboBox" name="left_click_combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="middle_click_label">
              <property name="text">
               <string>M&amp;iddle click</string>
              </property>
              <property name="buddy">
               <cstring>middle_click_combo</cstring>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QComboBox" name="middle_click_combo"/>
            </item>
            <item row="3" column="2">
             <widget class="Gui::Pref::TComboBox" name="double_click_combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="double_click_label">
              <property name="text">
               <string>&amp;Double click</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>double_click_combo</cstring>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLabel" name="xbutton2_label">
              <property name="text">
               <string>X button &amp;2</string>
              </property>
              <property name="buddy">
               <cstring>xbutton2_click_combo</cstring>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="xbutton1_label">
              <property name="text">
               <string>X button &amp;1</string>
              </property>
              <property name="buddy">
               <cstring>xbutton1_click_combo</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="3" rowspan="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="1" colspan="3">
             <widget class="QCheckBox" name="wait_for_double_click_check">
              <property name="text">
               <string>&amp;Wait for double click before triggering left click</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="wheel_group">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="title">
          <string>Wheel</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="wheel_label">
              <property name="text">
               <string>&amp;Current action</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>wheel_function_combo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="wheel_function_combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="wheel_actions_list_group">
            <property name="title">
             <string>Actions to cycle through when using the next wheel action</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="wheel_function_seek">
               <property name="text">
                <string>Media &amp;seeking</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="wheel_function_volume">
               <property name="text">
                <string>&amp;Volume control</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="wheel_function_zoom">
               <property name="text">
                <string>&amp;Zoom video</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="wheel_function_speed">
               <property name="text">
                <string>&amp;Change speed</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="wheel_function_seeking_reverse_check">
            <property name="text">
             <string>R&amp;everse wheel while seeking</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::Pref::TComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/pref/combobox.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::Action::TActionsEditor</class>
   <extends>QWidget</extends>
   <header>gui/action/actionseditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>input_tabwidget</tabstop>
  <tabstop>actions_editor</tabstop>
  <tabstop>left_click_combo</tabstop>
  <tabstop>double_click_combo</tabstop>
  <tabstop>wait_for_double_click_check</tabstop>
  <tabstop>right_click_combo</tabstop>
  <tabstop>middle_click_combo</tabstop>
  <tabstop>xbutton1_click_combo</tabstop>
  <tabstop>xbutton2_click_combo</tabstop>
  <tabstop>wheel_function_combo</tabstop>
  <tabstop>wheel_function_seek</tabstop>
  <tabstop>wheel_function_volume</tabstop>
  <tabstop>wheel_function_zoom</tabstop>
  <tabstop>wheel_function_speed</tabstop>
  <tabstop>wheel_function_seeking_reverse_check</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
